tests/CMakeLists.txt

```diff
@@ -200,3 +200,21 @@ add_dependencies(check llvm-slicer)
 # --------------------------------------------------
 add_executable(ptset-benchmark ptset-benchmark.cpp)
 target_link_libraries(ptset-benchmark PRIVATE dganalysis dgpta)
+
+# --------------------------------------------------
+# value-relations-test
+# --------------------------------------------------
+include_directories(${CMAKE_SOURCE_DIR}/include)
```

Comment:
This line is already present in this file.

Reply:
Sorry, I accidentally selected more lines than I intended. The comment above refers to line 207 only.